modules/Core/assets/ui/ingame/animationTest.ui

```diff
@@ -0,0 +1,77 @@
+{
+    "type": "Core:AnimationScreen",
```

Comment:
You don't need `Core:` in front here. It is a type definition not a reference to something else. Since the item is in Core it'll automatically work when referenced from elsewhere as `Core:animationTest`